extensions/resource-deployment/package.json

```diff
@@ -265,7 +265,7 @@
               },
               {
                 "name": "azdata-old",
```

Comment:
one day we should change this to something more descriptive 😄

Reply:
One day I'm just going to remove it in favor of the "new" azdata tool which relies on the azdata extension! 😁